api/database/migrations/2015_09_14_132023_create_rbac_tables.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Spira framework.
+ *
+ * @link https://github.com/spira/spira
+ *
+ * For the full copyright and license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRbacTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('auth_item', function (Blueprint $table) {
+            $table->string('name', 128)->primary();
+            $table->integer('type')->index();
```

Comment:
Should this be enum? If it is just micro-optimisation it is overkill and unclear what the intent is when viewing the database.